HCore-Database/Configuration/DatabaseEntityFrameworkCoreExtensions.cs

```diff
@@ -32,13 +36,26 @@ public static DbContextOptionsBuilder AddSqlDatabase(this DbContextOptionsBuilde
             }
             else
             {
-                builder.UseNpgsql(connectionString, options =>
-                {                   
-                    if (!string.IsNullOrEmpty(migrationsAssembly))
-                        options.MigrationsAssembly(migrationsAssembly);
+                var dataSource = _sqlDataSourcesByConfigurationKey.GetOrAdd(
```

Comment:
Npgsql datasources should be Singleton and since we don't have access to the container here I'm using a concurrent dictionary

Reply:
But why should they be created multiple times?